### PR TITLE
mvcc: add next meta index

### PR DESCRIPTION
**Pull request by disksing**

### CODE REVIEW

**Comment by disksing** [comment]
PTAL @ngaut @BusyJay

**Comment by BusyJay** [comment]
LGTM

File: proto/mvccpb.proto [hunk 1]
```diff
@@ -18,6 +18,7 @@ message MetaLock {
 }
 
 message Meta {
-    optional MetaLock lock        = 1;
-    repeated MetaItem items       = 2;
+    optional MetaLock lock  = 1;
+    repeated MetaItem items = 2;
+    optional uint64   next  = 3;
```
Comment (ngaut) [inline proto/mvccpb.proto]: Please add comments for next

**Comment by ngaut** [comment]
LGTM

**Pull request merged**